### ISSUE TITLE
Parallell steps nested inside a container step no longer show up in UI

### ISSUE THREAD

**Issue by flosell** [flosell/lambdacd#47]
LambdaCD seems to hide parallel nested steps if they are nested inside a sequential container step. 

To reproduce: Modify the given example pipeline by wrapping an in-parallel step in a `run`: 

```clojure
 (run
      (in-parallel
        ;; these child steps do some actual work with the checked out git repo
        (with-frontend-git
          create-some-details
          client-package)
        (with-backend-git
          server-test
          server-package)))
```

In this case, they still get executed as expected but don't show up in the UI: 
<img width="158" alt="screen shot 2015-09-12 at 18 30 51" src="https://cloud.githubusercontent.com/assets/540525/9832486/832dc6e4-597c-11e5-9cbf-3c3ae4efd144.png">

Workaround: This bug seems to have been introduced in version 0.5.1 so reverting back to 0.5.0 should resolve this issue.

Thanks to @redvox for finding this

**Comment by flosell** [comment]
Turns out this bug was a bit broader than initially thought: It happened in all cases where further nesting steps were the only children of a nesting step, e.g. 
```clojure
(foo
  (bar step-1)
  (baz step-2))
```

It's fixed now and a new release will be coming up soon.

**Comment by flosell** [comment]
Released in version [0.5.3](https://github.com/flosell/lambdacd/releases/tag/0.5.3)
